scripts/deleteAndLoadSnapsets.js

```diff
@@ -0,0 +1,398 @@
+/*!
+Copyright 2018 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script modifies the preferences database:
+// 1. Retrieves all the Prefs Safes of type "snapset" (prefsSafesType = "snapset") from the databsse.
+// 2. Retrieves all the GPII Keys associated with each snapset Prefs Safe so found,
+// 3. Deletes these Prefs Safes and their associated GPII Keys from the database,
+// 4. Uploads the new Prefs Safes and their GPII Keys to the database,
+// 5. Uploads demo user Prefs Safes and their Keys in to the database, if they are not already present.
+// A sample command that runs this script:
+// node deleteAndLoadSnapsets.js $COUCHDBURL $BUILD_DATA_DIR $BUILD_DEMOUSER_DIR
+
+"use strict";
+
+var http = require("http"),
+    url = require("url"),
+    fs = require("fs"),
+    fluid = require("infusion");
+
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.dataLoader");
+fluid.setLogging(fluid.logLevel.INFO);
+
+// Handle command line
+if (process.argv.length < 5) {
+    fluid.log("Usage: node deleteAndLoadSnapsets.js $COUCHDB_URL $BUILD_DATA_DIR $BUILD_DEMOUSER_DIR [--justDelete]");
+    process.exit(1);
+}
+
+var dbLoader = gpii.dataLoader;
```

Comment:
Please remove this global which just obstructs readers of the file

Reply:
What I really intended was:
var dbLoader = fluid.registerNamespace("gpii.dataLoader");

See, for comparison (where "windows" is used instead of "gpii.windows").
https://github.com/GPII/windows/blob/master/gpii/node_modules/WindowsUtilities/WindowsUtilities.js#L25

However, I'll use the full namespace going forward.